testsuite/tests/kuadrant/authorino/authconfig/conftest.py

```diff
@@ -0,0 +1,170 @@
+from testsuite.utils import ContentType
+
+
+@pytest.fixture(scope="session")
+def admin_rhsso(request, testconfig, blame):
+    """RHSSO OIDC Provider fixture"""
+    try:
+        testconfig.validators.validate(only="rhsso")
+        cnf = testconfig["rhsso"]
+        info = RHSSO(cnf["url"], cnf["username"], cnf["password"], blame("realm"), blame("client"),
+                     cnf["test_user"]["username"], cnf["test_user"]["password"])
+
+        if not testconfig["skip_cleanup"]:
+            request.addfinalizer(info.delete)
+
+        info.commit()
+        return info
+    except KeycloakAuthenticationError:
+        return pytest.skip("Unable to login into SSO, please check the credentials provided")
+    except KeyError as exc:
+        return pytest.skip(f"SSO configuration item is missing: {exc}")
+
+
+@pytest.fixture()
+def admin_auth(admin_rhsso):
+    """Returns RHSSO authentication object for HTTPX"""
+    return HttpxOidcClientAuth(admin_rhsso.get_token, "authorization")
+
+
+@pytest.fixture()
+def auth(rhsso):
+    """Returns RHSSO authentication object for HTTPX"""
+    return HttpxOidcClientAuth(rhsso.get_token, "authorization")
+
+
+@pytest.fixture(scope="module")
+def terms_and_conditions(request, mockserver, module_label):
+    """Creates Mockserver Expectation that returns whether terms are required and returns its endpoint"""
+
+    def _terms_and_conditions(value):
+        return mockserver.create_expectation(f"{module_label}-terms", f"/{module_label}/terms-and-conditions",
+                                             {"terms_required": value}, ContentType.APPLICATION_JSON)
+
+    request.addfinalizer(lambda: mockserver.clear_expectation(f"{module_label}-terms"))
+    return _terms_and_conditions
+
+
+@pytest.fixture(scope="module")
+def cluster_info(request, mockserver, module_label):
+    """Creates Mockserver Expectation that returns client ID and returns its endpoint"""
+
+    def _cluster_info(value):
+        return mockserver.create_expectation(f"{module_label}-cluster", f"/{module_label}/cluster-info",
+                                             {"client_id": value}, ContentType.APPLICATION_JSON)
+
+    request.addfinalizer(lambda: mockserver.clear_expectation(f"{module_label}-cluster"))
+    return _cluster_info
+
+
+@pytest.fixture(scope="module")
+def resource_info(request, mockserver, module_label):
+    """Creates Mockserver Expectation that returns info about resource and returns its endpoint"""
+
+    def _resource_info(middle_name, owner):
+        return mockserver.create_expectation(f"{module_label}-resource", f"/{module_label}/resource-info",
+                                             {"middle_name": middle_name, "owner": owner},
+                                             ContentType.APPLICATION_JSON)
+
+    request.addfinalizer(lambda: mockserver.clear_expectation(f"{module_label}-resource"))
+    return _resource_info
+
+
+@pytest.fixture()
+def commit():
```

Comment:
Why do you need to override the commit? Does the override even work? Because I don't know here you actually commit `authorization`

Reply:
Because I am rewriting the authorization fixture and using `openshift.new_app()` function for creation.